orko-app/pom.xml

```diff
@@ -79,6 +79,65 @@
         </dependency>
       </dependencies>
     </profile>
+    <profile>
+      <id>docker</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-shade-plugin</artifactId>
+            <executions>
+              <execution>
+                <id>shade</id>
+                <phase>none</phase>
+                <goals>
+                  <goal>shade</goal>
+                </goals>
+              </execution>
+            </executions>
+          </plugin>
+          <plugin>
+            <groupId>com.google.cloud.tools</groupId>
+            <artifactId>jib-maven-plugin</artifactId>
+            <version>1.2.0</version>
+            <configuration>
+              <to>
+                <image>orko:latest</image>
+              </to>
+              <container>
+                <jvmFlags>
+                  <jvmFlag>-XX:+UnlockExperimentalVMOptions</jvmFlag>
+                  <jvmFlag>-XX:+UseCGroupMemoryLimitForHeap</jvmFlag>
+                  <jvmFlag>-Dsun.net.inetaddr.ttl=60</jvmFlag>
+                  <jvmFlag>-Dio.netty.leakDetectionLevel=advanced</jvmFlag>
+                  <jvmFlag>-Dmorf.mysql.noadmin=true</jvmFlag>
+                  <jvmFlag>-Ddw.server.connector.port=8080</jvmFlag>
+                  <jvmFlag>-Dcom.sun.management.jmxremote.rmi.port=9090</jvmFlag>
+                  <jvmFlag>-Dcom.sun.management.jmxremote=true</jvmFlag>
+                  <jvmFlag>-Dcom.sun.management.jmxremote.port=9090</jvmFlag>
+                  <jvmFlag>-Dcom.sun.management.jmxremote.ssl=false</jvmFlag>
+                  <jvmFlag>-Dcom.sun.management.jmxremote.authenticate=false</jvmFlag>
+                  <jvmFlag>-Dcom.sun.management.jmxremote.local.only=false</jvmFlag>
+                  <jvmFlag>-Djava.rmi.server.hostname=localhost</jvmFlag>
+                </jvmFlags>
+                <mainClass>com.gruelbox.orko.app.monolith.MonolithApplication</mainClass>
```

Comment:
Can this be modified to use `base-docker-config.yml` inside if nothing else is specified?